vmdb/lib/miq_automation_engine/service_models/miq_ae_service_service_orchestration.rb

```diff
@@ -0,0 +1,38 @@
+module MiqAeMethodService
+  class MiqAeServiceServiceOrchestration < MiqAeServiceService
+    expose :orchestration_template
+    expose :orchestration_manager
+    expose :stack_name
+    expose :stack_name=
+    expose :stack_options
+    expose :stack_options=
+    expose :orchestration_stack_status
+    expose :deploy_orchestration_stack
+
+    def orchestration_template=(template)
+      ar_method do
+        if template.nil?
+          @object.orchestration_template = nil
+        elsif template.kind_of?(MiqAeMethodService::MiqAeServiceOrchestrationTemplate)
+          @object.orchestration_template = OrchestrationTemplate.find(template.id)
+        else
+          raise ArgumentError, "template must be a MiqAeServiceOrchestrationTemplate"
+        end
+        @object.save
+      end
+    end
```

Comment:
What do you think of something like this?  I like having the raise line at the to the top of the method.  Could do the same treatment to `orchestration_manager=` as well.
```ruby
def orchestration_template=(template)
  raise ArgumentError, "template must be a MiqAeServiceOrchestrationTemplate or nil" if template && !template.kind_of?(MiqAeMethodService::MiqAeServiceOrchestrationTemplate)

  ar_method do
    @object.orchestration_template = template ? OrchestrationTemplate.where(:id => template.id) : nil
    @object.save
  end
end
```